modules/fbs/fbs.module

```diff
@@ -520,3 +623,22 @@ function _fbs_wayf_login($name, $value = NULL) {
 
   return NULL;
 }
+
+/**
+ * Implements hook_ting_pre_execute().
+ *
+ * Filter out blacklisted branches from search.
+ */
+function fbs_ting_pre_execute($request) {
+  // We only want to modify searches.
+  if (gettype($request) != 'TingClientSearchRequest') {
+    return;
+  }
```

Comment:
Will gettype work here? Look at possible values in documentation: http://php.net/manual/en/function.gettype.php

Think it's better to use is_a or instanceof.

Also are we targeting TingCollectionRequest also? If not get_class should be used instead.